src/main/java/com/google/sps/utility/GeocodingResultUtility.java

```diff
@@ -18,57 +18,79 @@
 import com.google.maps.model.GeocodingResult;
 import com.google.maps.model.LatLng;
 import com.google.maps.model.PlaceType;
+import java.util.Arrays;
+import java.util.List;
 import java.util.Optional;
 
 /** Utility class to extract data from GeocodingResult objects. */
 public class GeocodingResultUtility {
   /**
-   * Parses for coordinates in a resulting call to the Geocoding API.
+   * Parses for the first coordinate found in a resulting call to the Geocoding API.
    *
```

Comment:
I think it's worth mentioning your `AddressType.STREET_ADDRESS` check because when you find a match, you may end up not returning the first coordinate.

Reply:
Good point, thanks!

Reply:
`RESOLVED`